### PR TITLE
Add workflow articles for version 10

### PR DESCRIPTION
**Pull request by eshanrnh**

### CODE REVIEW

**Review by nathanwoulfe — APPROVED** [review]
Hey @eshanrnh, looks good to me - couple of refs to v11 that should be v10, but the startup stuff is spot on.

File: umbraco-workflow/installing-workflow.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+* [Visual Studio Installation](installing-workflow.md#visual-studio-installation)
+
+### .Net CLI Installation
+
+To install the Umbraco Workflow package (Umbraco.Workflow), follow these steps:
+
+1. Run the following command to add a package reference to your Umbraco project:
+
+    ```
+    dotnet add package Umbraco.Workflow --version 11.0.0
```
Comment (sofietoft) [inline umbraco-workflow/installing-workflow.md]: Version number needs to be updated here

File: umbraco-workflow/installing-workflow.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+3. Browse for **Umbraco.Workflow**.
+4. Select the appropriate version from the Version drop-down depending on the Umbraco version you are using.
+5. Click Install.
+
+    <figure><img src="images/VS_Installation.png" alt=""><figcaption></figcaption></figure>
+6. Once the package is installed, open the **.csproj** file to make sure the package reference is added:
+
+    ```xml
+    <ItemGroup>
+    <PackageReference Include="Umbraco.Workflow" Version="11.0.0" />
```
Comment (sofietoft) [inline umbraco-workflow/installing-workflow.md]: Version number need to be updated here

File: umbraco-workflow/migrating-workflow.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Umbraco Workflow is backwards compatible with Plumber data. However, with a new default namespace constituting a major breaking change any existing customization or extension needs to be updated.
+
+To migrate from an Umbraco installation with an existing Plumber installation to Umbraco Workflow, follow these steps:
+
+{% tabs %}
+{% tab title="Site running on SQL Server database" %}
+1. Uninstall Plumber and remove the `/App_Plugins/Plumber` folder.
+2. Upgrade your project to Umbraco 11.
+3. Install Umbraco Workflow 11. See the [Installing Umbraco Workflow](installing-workflow.md) article.
```
Comment (sofietoft) [inline umbraco-workflow/migrating-workflow.md]: Version numbers are wrong

File: umbraco-workflow/migrating-workflow.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+{% endhint %}
+{% endtab %}
+
+{% tab title="Site running on SQLite database" %}
+1. Uninstall Plumber and remove the `/App_Plugins/Plumber` folder.
+2. Upgrade your project to Umbraco 11.
+3. Make a copy of the `Value` column from the `WorkflowSettings` table.
+4. Delete the `WorkflowSettings` table.
+5. Update `WorkflowTaskInstance` table to allow null values in the GroupId column.
+6. Install Umbraco Workflow 11. See the [Installing Umbraco Workflow](installing-workflow.md) article.
```
Comment (sofietoft) [inline umbraco-workflow/migrating-workflow.md]: Version numbers here as well!

File: umbraco-workflow/upgrading-workflow.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* Select **Umbraco.Workflow**.
+*   Select the latest version from the **Version** drop-down and click **Install**.
+
+
+    ![NuGet Package Manager](images/Manage-packages-v11.png)
+*   Open the **.csproj** file to make sure the package reference is updated:
+
+    ```xml
+    <ItemGroup>
+      <PackageReference Include="Umbraco.Workflow" Version="11.x.x" />
```
Comment (sofietoft) [inline umbraco-workflow/upgrading-workflow.md]: Version 11 reference

**Review by sofietoft — COMMENTED** [review]
I've found (hopefully) all Umbraco 11 and Workflow 11 references.
They should be updated to 10, right?

**Review by sofietoft — APPROVED** [review]
LGTM 💪

**Comment by eshanrnh** [comment]
Thanks for the review, @nathanwoulfe and @sofietoft 💪🙌 Merging the PR in.

**Pull request merged**